packages/package_searchgui_1_26/tool_dependencies.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="searchgui" version="1.26">
+        <install version="1.0">
+            <actions>
+                <action type="download_by_url" target_filename="SearchGUI-1.26.2.tar.gz">http://genesis.ugent.be/maven2/eu/isas/searchgui/SearchGUI/1.26.2/SearchGUI-1.26.2-mac_and_linux.tar.gz </action>
+                <action type="shell_command">chmod -R o+w resources</action>
```

Comment:
Yes it will give `others` write access. I think this is a left over from the original installation instruction. The user who is running this installation should also execute the jobs ... we should remove this line and see if it still works if you think this can be problematic. In theory this should work :)